### DISCUSSION TITLE
Any way to rerun cell whenever a file changes?

### DISCUSSION THREAD

**Discussion by samuela** [fonsp/Pluto.jl#1368] [Questions about using Pluto] [answered]
I use Pluto in conjunction with other scripts and files. It would be a game-changer for my workflow if there was a way to trigger the re-evaluation of a cell by watching some file for changes. Is this possible today? If not, what would it involve to make it a reality?

**Comment by Pangoraw** [comment]
Unfortunately, the APIs are not there yet to do this from inside Pluto. Thinks like [triggering a bond update](https://github.com/fonsp/Pluto.jl/pull/991#issuecomment-796837127) will help. In the meantime if you want a (hacky) solution, I made [PlutoBindClient.jl](https://github.com/Pangoraw/PlutoBindClient.jl) to trigger a bond update from an external script. There is an example where it sends Github event to a notebook [here](https://github.com/Pangoraw/PlutoBindClient.jl/blob/9648777b10e0a99171b3323ad2b38ad21708b6db/examples/Github.jl#L17).

**Comment by Pangoraw** [comment, marked as answer]
I just stumbled onto that discussion and now you can use [PlutoLinks.jl](https://github.com/JuliaPluto/PlutoLinks.jl) `@use_file(filename)` macro which does just that and returns the updated file content.

```julia
using PlutoLinks
#==#
file_content = @use_file("path_to_file.txt")
#==#
do_something_with(file_content)
```
